### DISCUSSION TITLE
My PIN dont work

### DISCUSSION THREAD

**Discussion by paugalmees** [iotaledger/firefly#2927] [Help & Support]
I can't enter my wallet because it says that my PIN is wrong. Im cheking and im putting it good

**Comment by t00k** [comment]
On the login screen, add a new profile, but recover from the SAME 24 words/stronghold backup/ledger nano.

For some users, especially on Windows the credentials manager was wiped during an update, which gives this situation.

**Reply by Osscaree** [reply]
Eu fiz um novo perfil com minhas 24 palavras originais , porem como recupero as moedas da carteira anterior?

**Reply by Osscaree** [reply]
Passada uma hora os fundos se atualizaram
